src/main/java/com/direwolf20/buildinggadgets/client/gui/NameDisplayData.java

```diff
@@ -0,0 +1,33 @@
+package com.direwolf20.buildinggadgets.client.gui;
+
+final class NameDisplayData {
```

Comment:
Is this class going to be used more than once because currently it's only being used once and could be argued that it's not needed.

Reply:
Personally I'm not completely sure, however it is better than using a int array to store data. With a int array we hacked it with 1's and 0's, which painful to read (what in the world is `data[3]`?).